atddiff/test/test_new.atd

```diff
@@ -0,0 +1,75 @@
+(*
```

Comment:
Ideally this would split in different test files for each scenario and each file could contain 3 sections:
old_atd content
new_atd_content
expected output of atdiff

so you see everything in one file instead of now you need to inspect 3 separate files to understand.
But not super important. More important to get v0 out of the door!

Reply:
It turns out dune doesn't support dynamic targets. I created a shell script that gathers test files and generates a `dune` file. I wish there was a simpler and cleaner way.

Anyway, now it's easy to split the test files.